package.json

```diff
@@ -10,7 +10,8 @@
 			"apps/*",
 			"packages/*",
 			"test/e2e",
-			"test/e2e/lib/reporter"
+			"test/e2e/lib/reporter",
```

Comment:
Nesting packages is somewhat unreliable in `yarn`, I'd recommend extracting it to a separate package inside `./packages`

Reply:
Actually none of `test/e2e/lib/reporter` or `test/visual` seem to be packages. What's the goal of declaring them here?

Reply:
Ah, I originally has a separate package.json in the visual directory. This was a leftover.